jestScaffolding.js

```diff
@@ -17,13 +16,28 @@ const customMatchers = [
       name: 'toHaveBeenCalledOnceWith',
       receives: 'jest.fn()',
     },
-    matcher: (expectChain, expected) =>
-      expectChain.toHaveBeenCalledTimes(1).toHaveBeenCalledWith(...expected),
+    matcher: (expectChain, expected) => {
+      expectChain.toHaveBeenCalledTimes(1);
+      expectChain.toHaveBeenCalledWith(...expected);
+    },
     negationDiff: (extendApi, _, expected) =>
       `Expected spy not to have been called ${extendApi.utils.RECEIVED_COLOR(
         'once',
       )} with ${extendApi.utils.printReceived(expected)}`,
   },
+  {
+    description: {
+      name: 'toBeRejectedWith',
+    },
+    matcher: async (expectChain, [expected]) => {
+      expect.hasAssertions();
```

Comment:
This is a cool `jest` functionality that verifies that a test case invokes at least one matcher! It's very handy in async matchers, like this one. If we forget to `return`/`await` them, they will be invoked after the test case has run and this could result in a false positive result.